awswrangler/typing.py

```diff
@@ -165,6 +165,29 @@ class AthenaPartitionProjectionSettings(TypedDict):
     """
 
 
+class RayModinSettings(TypedDict):
+    """Typed dictionary defining the settings for distributing calls using Ray/Modin."""
+
+    parallelism: NotRequired[int]
+    """
+    The requested parallelism of the read.
+    Parallelism may be limited by the number of files of the dataset.
+    Auto-detect by default.
+    """
+
+
+class RayModinReadSettings(RayModinSettings):
```

Comment:
Perhaps the class above should be `RayReadSettings` and this one `RayReadParquetSettings`?